runtime/port/sysvipc/j9shmem.c

```diff
@@ -1348,10 +1348,14 @@ j9shmem_createDir(struct J9PortLibrary* portLibrary, char* cacheDirName, uintptr
 		Trc_PRT_j9shmem_createDir_ExitFailedFile();
 		break;
 	case EsIsDir:
-#if !defined(J9ZOS390)
-		if (J9SH_DIRPERM_ABSENT == cacheDirPerm) {
+#if !defined(J9ZOS390) && !defined(OPENJ9_BUILD)
```

Comment:
Why are we introducing new `!OPENJ9_BUILD` code here?  Why wouldn't OpenJ9 update the directory permissions in this case?

Reply:
If user created a directory and set permission for that directory. I think we shouldn't overwrite the permission in this case.

Reply:
I guess my question is - why is this behaviour different between builds that define OPENJ9_BUILD and anything else (IBM Java?)?

Reply:
I am fine with keeping the behavior here consistent regardless of OPENJ9_BUILD. 
Then we need to choose whether we want to overwrite the user's permission setting on an existing directory. I preference is not to overwrite.

Reply:
Code updated to make the behavior consistent on existing directory.